Cargo.toml

```diff
@@ -466,10 +466,8 @@ jemallocator = { version = "0.3.2", features = [
 ] }
 json-patch = "0.2.6"
 jsonwebtoken = "8.1"
-k8s-openapi = { version = "0.11.0", default-features = false, features = [
-    "v1_15",
-] }
-kube = { version = "0.51.0", features = ["jsonpatch"] }
+k8s-openapi = { version = "0.13.1", default-features = false, features = ["v1_22"] }
```

Comment:
So if you try to include this from another binary that has this dep, it will fail since the v1_22 flag cannot be enabled in several places

I tried to fix this to no avail, instead actually I suggest moving the k8s deps to the forge binary Cargo toml

Reply:
Sounds good, moved these dependencies to forge crate.